hack/integration-benchmarks.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
for consistency with our other script names, I think this one should be benchmark-integration.sh?